remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/indexing/IndexingMetadataCreator.java

```diff
@@ -55,12 +59,12 @@ public IndexingMetadata create(Descriptor descriptor, AnnotationElement.Annotati
          }
 
          boolean isSortable = false;
-         AnnotationElement.Annotation sortableFieldAnnotation = fd.getAnnotations().get(IndexingMetadata.SORTABLE_FIELD_ANNOTATION);
+         AnnotationElement.Annotation sortableFieldAnnotation = findAnnotation(fd.getAnnotations(), IndexingMetadata.SORTABLE_FIELD_ANNOTATION);
          if (sortableFieldAnnotation != null) {
             isSortable = true;
          }
```

Comment:
except if I'm missing something, this could be replaced with
```java
boolean isSortable = findAnnotation(fd.getAnnotations(), IndexingMetadata.SORTABLE_FIELD_ANNOTATION) != null;
```

Reply:
Done